packages/order-utils/test/exchange_transfer_simulator_test.ts

```diff
@@ -42,6 +42,7 @@ describe('ExchangeTransferSimulator', async () => {
             from: devConstants.TESTRPC_FIRST_ADDRESS,
         };
 
+        await blockchainLifecycle.startAsync();
```

Comment:
This was affecting the new unit tests in `order_validation_utils_test.ts` that are also blockchain dependent.